objects/Cargo.toml

```diff
@@ -21,8 +21,9 @@ bench = false
 [features]
 default = ["std"]
 std = ["assembly/std", "crypto/std", "miden-core/std", "miden-lib/std", "miden-processor/std", "miden-verifier/std"]
-testing = ["miden-test-utils"]
+testing = ["miden-test-utils", "builder"]
 serde = ["dep:serde", "crypto/serde"]
+builder = ["dep:rand"]
```

Comment:
nit: should we call this feature "mock" or something like that?

Reply:
changed :)